### PR TITLE
Use SVG logo

### PR DESCRIPTION
**Pull request by LitoMore**
### Preview

|||
|:-:|:-:|
|**Before**||
|**After**||

### Additional comments

I am making this logo for your icon request at https://github.com/simple-icons/simple-icons/pull/7702. Then you could recreate the icon base on this one.

I used Photoshop and Sketch for recreating the logo. And compressed the SVG file with SVGO. And the font is `Arial Bold`. Hope everything are correct.

image

### CODE REVIEW

**Comment by motdotla** [comment]
🙇‍♂️ to your skills. Solid!

File: .npmignore [hunk 1]
```diff
@@ -1,6 +1,7 @@
 .nyc_output/
 examples/
 flow-typed/
+media/
```
Comment (motdotla) [inline .npmignore:4]: Rather than in media, please place in root of the project next to dotenv.png.

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 # dotenv
 
-<img src="https://raw.githubusercontent.com/motdotla/dotenv/master/dotenv.png" alt="dotenv" align="right" />
+<img src="https://raw.githubusercontent.com/motdotla/dotenv/master/media/dotenv.svg" alt="dotenv" align="right" />
```
Comment (motdotla) [inline README.md:5]: make this `/dotenv.svg` rather than `/media/dotenv.svg`

**Review by motdotla — CHANGES_REQUESTED** [review]
Bring back the dotenv.png file. Many many people and websites depend on that already - they link directly to it over GitHub's raw cdn service.

Once those changes are done, this is ready to go.

File: .npmignore (hunk 1 above)
Comment (LitoMore) [reply .npmignore]: np.

File: README.md (hunk 2 above)
Comment (LitoMore) [reply README.md]: Done.

**Review by motdotla — APPROVED** [review]

**Comment by LitoMore** [comment]
@motdotla Dotenv is available on [shields.io](https://shields.io/#styles) now!

**Pull request merged**